build.gradle

```diff
@@ -191,15 +191,19 @@ task download(type: Copy) {
 	into 'src/jar'
 }
 
-task gitRevList() {
+task getRevision() {
 	onlyIf {
 		file('.git').exists()
 	}
 	inputs.dir('.git')
 	outputs.files file('build/revision.txt')
 
 	doLast {
-		def revision = grgit.log(includes:['origin/main']).size()
+		def remote = System.getenv('HEAD_COMMIT_HASH')
```

Comment:
@Rinn is this sufficient for your use case? I'm not sure how else to do this.

Reply:
I'll prep a PR separately for this after you commit this, there's some gradle stuff to consider the environment variable as a dependency so it invalidates the task correctly.

Reply:
#72 should do it, but I'll wait for you to commit this first.